main/src/ca/uwaterloo/flix/api/lsp/provider/completion/CompletionUtils.scala

```diff
@@ -143,4 +143,9 @@ object CompletionUtils {
     }
   }
 
+  /**
+    * Checks if we are ready to offer AutoUseCompletion or AutoImportCompletion.
+    * Currently, we will only offer them if at least three characters have been typed.
+    */
+  def readyToComplete(word: String): Boolean = word.length >= 3
```

Comment:
I think the word "ready" is not exactly right. How about something like: "shouldComplete" or "includeCompletion"?

Reply:
vote for shouldComplete